scalatest/src/main/scala/org/scalatest/tools/DashboardReporter.scala

```diff
@@ -646,7 +646,7 @@ private[scalatest] class DashboardReporter(directory: String,
   //
   class SuiteRecord(startEvent: SuiteStarting) {
     var nestedElements = List[Any]()
-    var endEvent: Event = null
+    var endEvent: Event | Null = null
```

Comment:
**Nullable variable**
Variables that really can be null and should therefore have a `| Null` type.